package.json

```diff
@@ -118,16 +117,16 @@
     "eslint-plugin-react": "7.24.0",
     "gh-release": "6.0.0",
     "git-semver-tags": "4.1.1",
-    "husky": "6.0.0",
-    "jest": "26.6.3",
+    "husky": "7.0.1",
+    "jest": "^26.6.3",
```

Comment:
Can you make the dependencies exact before merging?

Reply:
Updated! I also updated `typescript` but I had to modify one test. It might be a little unstable. Can you check if this is ok?

Reply:
NVM